styles/styles.css

```
@@ -386,25 +519,64 @@ body {
   display: flex;
   justify-content: space-between;
   flex-wrap: wrap;
+  padding: 56px 40px;
 }
 
 .advantages-item {
-  width: 442px;
+  width: 505px;
   line-height: 22px;
-  padding: 0 40px;
+}
+
+.advantages-text {
+  position: relative;
+  margin-left: 64px;
+  color: #565c66;
+}
+
+.advantages-text::before {
+  content: " ";
```

```suggestion
  content: "";
```
пробел не нужен. Тут и везде исправь плиз